pydy/codegen/ode_function_generators.py

```diff
@@ -641,16 +641,26 @@ class CythonODEFunctionGenerator(ODEFunctionGenerator):
 
     def __init__(self, *args, **kwargs):
 
+        self._options = {'tmp_dir': None,
+                         'prefix': 'pydy_codegen',
+                         'cse': True,
+                         'verbose': False}
+        for k, v in self._options.items():
+            self._options[k] = kwargs.pop(k, v)
+
```

Comment:
This works as a way to pull the arguments out of kwargs properly but ends up setting each item in the `self._options` dictionary twice. Another way would be to just call kwargs for each item like this

```python
self._options = {}
self._options['tmp_dir'] = kwargs.pop('tmp_dir', None)
self._options['prefix'] = kwargs.pop('prefix', 'pydy_codegen')
self._options['cse'] = kwargs.pop('cse', True)
self._options['False'] = kwargs.pop('verbose', False)
```

This method probably a bit more verbose and thus readability is reduced but I imagine the performance would improve because each item in the dictionary is only being set once. What do you think?